packages/forma-36-tokens/src/tokens/colors/colors-contrast.js

```diff
@@ -1,7 +1,10 @@
+const colorsBlack = require('./colors-black');
+const colorsGray = require('./colors-gray');
+
 const colorsContrast = {
-  'color-contrast-dark': '#0C141C',
-  'color-contrast-mid': '#192532',
-  'color-contrast-light': '#263545',
+  'color-contrast-dark': colorsBlack['color-black'],
```

Comment:
are we going to deprecate these colors? or they'll be there but mapped to the scale colors?

Reply:
We would like to deprecate those colors and keep only `colors-gray`. 
I was wondering how to handle those cases. I guess we need to keep this file (most likely our clients use them?). So then I would need to do something like:
```
const colorsContrast = require('./colors-contrast-deprecated);

module.exports = colorsContrast;
```
And move this mapping into this file. Overkill or it makes sense?

Reply:
Ok, as discussed, I reexported colors in the main files and created additional `deprecated` files for them. 
Colors that we would keep are in `scale` files (then reexported in `colors-`) + `colors-semantic.js` all the rest will be deprecated.